webpack.config.js

```diff
@@ -108,6 +108,8 @@ const nodeModulesToTranspile = [
 	'd3-array/',
 	'd3-scale/',
 	'debug/',
+	'@wordpress/',
+	'react-dates',
```

Comment:
I think that `react-dates/` is better because otherwise we match also `react-dateselector` etc?

Typically we transpile NPM dependencies because they contain ES6 syntax. I assume that in this case we only want to modify the Lodash imports?

Reply:
That's right; we want to avoid this package dragging in `lodash` as a dependency. I'll double-check if we still need it, and if so I'll make the change you suggested.

Reply:
Looks like this line is no longer needed, since `react-dates` is no longer part of the bundles (`@wordpress/components` was the one using it, and that seems to no longer be part of the Calypso bundles?)